simulator_service/srv/SimulateTrajectory.srv

```diff
@@ -0,0 +1,4 @@
+trajectory_msgs/JointTrajectory trajectory
+bool wait_for_execution
```

Comment:
While I agree, my goal here was to model the existing API of MoveIt: see [here](http://docs.ros.org/jade/api/moveit_msgs/html/srv/ExecuteKnownTrajectory.html). I'll add a comment.